src/ray/core_worker/test/direct_task_transport_test.cc

```diff
@@ -806,6 +806,8 @@ TEST(DirectTaskTransportTest, TestConcurrentWorkerLeases) {
   auto task_finisher = std::make_shared<MockTaskFinisher>();
   auto actor_creator = std::make_shared<MockActorCreator>();
   auto lease_policy = std::make_shared<MockLeasePolicy>();
+
+  int64_t concurrency = 10;
```

Comment:
nit: use consistent types, really linter should catch signed->unsigned implicit conversion.